src/dream.mli

```diff
@@ -1746,11 +1746,12 @@ val graphiql : ?default_query:string -> string -> handler
     {{:https://cheatsheetseries.owasp.org/cheatsheets/Database_Security_Cheat_Sheet.html}
     OWASP {i Database Security Cheat Sheet}}. *)
 
-val sql_pool : ?size:int -> string -> middleware
+val sql_pool : ?size:int -> ?post_connect:((module Caqti_lwt.CONNECTION) -> (unit, Caqti_error.t) result Lwt.t) -> string -> middleware
```

Comment:
The Dream interface uses `promise` instead of `Lwt.t`. Also note the line wrapping is set to around 80 characters and function signatures are wrapped breaking at the end of the `->` symbol.